relay-server/src/envelope.rs

```diff
@@ -38,15 +38,16 @@ use std::io::{self, Write};
 use bytes::Bytes;
 use chrono::{DateTime, Utc};
 use failure::Fail;
+use relay_common::DataCategory;
```

Comment:
_nit_: Please move this import to the next group where all the `relay_*` imports are.